app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @ingredients = @recipe.find_ingredients
+    @cost = @recipe.find_cost
```

Comment:
Same thing here. I would just call your `find_cost` method in your view on `@recipe`. Remember we want to keep our controllers slim.